src/main/py/hl7_listener/main.py

```diff
@@ -44,10 +44,13 @@ async def process_received_hl7_messages(hl7_reader, hl7_writer):
         hl7_message = None
         while not hl7_reader.at_eof():
             hl7_message = await hl7_reader.readmessage()
-            logger.info(logging_codes.HL7_MLLP_MSG_RECEIVED)
             # This may not be needed since the hl7_mllp sender should fail if the message
             # was not valid hl7 message.
-            hl7.parse(str(hl7_message))
+            _parsed = hl7.parse(str(hl7_message))
```

Comment:
We could move the `_parsed` and other `_` variables into the `if`. That appears to be the only place that they are used.

Reply:
We're actually logging them either way, whether or not the dd span has been injected via the decorator to the function's `ddspan` param.